.github/workflows/deploy-hubs.yaml

```diff
@@ -18,6 +18,7 @@ on:
       - "!deployer/health_check_tests/**"
       - "!deployer/commands/generate/billing/**"
       - "!deployer/commands/generate/dedicated_cluster/**"
+      - "!deployer/commands/generate/hub_asset/**"
```

Comment:
Not totally relevant to this PR but...

There's a comment a little further up that states "Include changes to the deployer script's folder, but exclude some parts" - it would be nice to have some more detail on what parts we _do_ want to include and what parts we don't, and _why_. Because my initial reaction seeing this change was "At what point do we just list `!deployer/**`?" and I'm still in the camp that the deployer should be a separate repo tbh.